pkg/utils/errors/unwrap.go

```diff
@@ -16,30 +16,18 @@ package errors
 
 import "errors"
 
-type causer interface {
-	Cause() error
-}
-
-// Unwrap unwraps and returns the root error. Multiple wrappings either via `fmt.Errorf` or via `causer` implementations are properly taken into account.
+// Unwrap unwraps and returns the root error. Multiple wrappings via `fmt.Errorf` implementations are properly taken into account.
 func Unwrap(err error) error {
-	var cdone, udone bool
+	var udone bool
```

Comment:
nit: can you rename `udone` variable, there's no reason for the weird name now.
```suggestion
	var done bool
```

Reply:
Done.